TriblerGUI/widgets/trustpage/js/drawing.js

```diff
@@ -121,6 +109,20 @@ function mouseInNode(node, infoLabel) {
         .style("top", node.y + "px");
 }
 
+function getTableContents(quantities) {
+    var tableRows = "";
+    for (var quantity in quantities)
+        tableRows.concat(createTableRow(quantity.key, quantity.value));
+    return tableRows;
+}
+
+function createTableRow(quantityDescription, quantityValue) {
+    return "<tr>" +
+        "<td class=\"metric\"> + quantityDescription + </td>" +
```

Comment:
@rkeulemans I believe there are some double quote's missing around `quantityDescription` and `quantityValue`.